extensions/amp-story/1.0/amp-story-share.js

```diff
@@ -277,13 +320,35 @@ export class ShareWidget {
     // TODO(alanorozco): Listen for proper tap event (i.e. fastclick)
     listen(linkShareButton, 'click', e => {
       e.preventDefault();
-      this.copyUrlToClipboard_();
+      this.copyUrlToClipboard_(this.root.querySelector('#page-share').checked);
     });
   }
 
-  /** @private */
-  copyUrlToClipboard_() {
-    const url = Services.documentInfoForDoc(this.getAmpDoc_()).canonicalUrl;
+  /** On desktop mode, with branching, users should be able to share a story
+   * starting from a specific page.
+   * @private */
+  maybeAddPageShareButton_() {
+    if (isExperimentOn(this.win, 'amp-story-branching') && this.isDesktopUi_) {
```

Comment:
Instead of keeping track of `this.isDesktopUi_`, would it make sense to just directly call `this.storeService_.get(StateProperty.UI_STATE) === UIType.DESKTOP_FULLBLEED` here?

Reply:
Yes, definitely. Made the change, and did the same with the `currentPageId`